tests/bearlib/languages/documentation/DocumentationCommentTest.py

```diff
@@ -36,7 +36,7 @@ def test_fields(self):
         self.assertEqual(uut.indent, ' ')
         self.assertEqual(str(uut), 'my doc')
         self.assertEqual(uut.marker, ('/**', '*', '*/'))
-        self.assertEqual(uut.range, (25, 45))
+        self.assertEqual(uut.position, (25, 45))
 
```

Comment:
Imo it shall just test if fields/properties are populated correctly from `__init__` calls.